src/mds/MDSCacheObject.h

```diff
@@ -70,55 +72,29 @@ class MDSCacheObject {
   static const int PIN_CLIENTLEASE = 1009;
   static const int PIN_DISCOVERBASE = 1010;
```

Comment:
Treat these as data members rather than constants to have a better arrangement.
Otherwise, looks good to me.

Reply:
Since we are following the same order in other header files. Let's not change it.